src/test/java/com/fasterxml/jackson/databind/deser/TestArrayOfFloats.java

```diff
@@ -0,0 +1,32 @@
+package com.fasterxml.jackson.databind.deser;
+
+import com.fasterxml.jackson.databind.BaseMapTest;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class TestArrayOfFloats extends BaseMapTest
+{
+
+    /*
+    /**********************************************************
+    /* Tests
+    /**********************************************************
+     */
+
+    public void testArrayOfFloats() throws Exception
+    {
+        ObjectMapper mapper = new ObjectMapper();
+        StringBuilder sb = new StringBuilder();
+        sb.append('[')
+                .append("\"7.038531e-26\",")
+                .append("\"1.199999988079071\",")
+                .append("\"3.4028235677973366e38\",")
+                .append("\"7.006492321624086e-46\"")
+                .append(']');
+        float[] floats = mapper.readValue(sb.toString(), float[].class);
```

Comment:
Maybe add test(s) for field of `float` and wrapper `Float` values too? Array deserializers are different from single-value ones.

Reply:
@cowtowncoder I added some extra scenarios